mosaic-site/src/components/app-header/AppHeader.tsx

```diff
@@ -0,0 +1,78 @@
+import React, { FC, useEffect, useLayoutEffect, useState } from "react";
+import classnames from "classnames";
+import { Logo } from "@salt-ds/lab";
+import { useBreakpoint, Link } from "@jpmorganchase/mosaic-components";
+import type { TabsMenu, TabsLinkItem } from "@jpmorganchase/mosaic-components";
+import { useRoute, SidebarItem } from "@jpmorganchase/mosaic-store";
+import {
+  AppHeaderDrawer,
+  AppHeaderTabs,
+} from "@jpmorganchase/mosaic-site-components";
+import styles from "./AppHeader.module.css";
+
+export type AppHeaderProps = {
+  homeLink?: string;
+  logo?: string;
+  menu?: TabsMenu;
+  title?: string;
+};
+
+const useIsomorphicLayoutEffect =
```

Comment:
Should we just import this from Salt?

Reply:
FYI @DavieReid @lilyvc

Reply:
> Should we just import this from Salt?

Unfortunately it won't work with `withAppHeaderAdapter`

Reply:
> > Should we just import this from Salt?
> 
> Unfortunately it won't work with `withAppHeaderAdapter`

What was the issue?

Reply:
It's to do with the way these components have been built, `useAppHeader()` from the mosaic store returns a `homeLink`, `logo`, `menu` and `title` whereas Salt's AppHeader only takes in `children` so we would have to rewrite all of this logic

Reply:
Oh sorry I meant useIsomorphicLayout specifically

Reply:
> Oh sorry I meant useIsomorphicLayout specifically

Sorry my bad! I've added this in